assets/src/components/dashboard/Overview.js

```
@@ -94,10 +94,10 @@ const Overview = () => {
                               encounter. We're just an email away!
                             </p>
                             <a
-                              href={baseUrlStoregrowth + links?.support}
+                              href={'https://storegrowth.io/contact-us/'}
                               target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)